pkg/server/container_create_unix.go

```diff
@@ -124,6 +124,7 @@ func (c *criService) containerSpec(id string, sandboxID string, sandboxPid uint3
 	if config.GetTty() {
 		specOpts = append(specOpts, oci.WithTTY)
 	}
+	specOpts = append(specOpts, oci.WithDefaultPathEnv)
```

Comment:
Move this after `WithoutRunMount`, that is where default should locate.

Reply:
And please add a unit test like https://github.com/containerd/cri/blob/master/pkg/server/container_create_unix_test.go#L735 to verify that PATH should not be empty by default. It can help us catch future issues.